internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -1118,6 +1118,8 @@ func newAssistedCM(ctx context.Context, log logrus.FieldLogger, asc ASC) (client
 			"JWKS_URL":                    "https://api.openshift.com/.well-known/jwks.json",
 			"PUBLIC_CONTAINER_REGISTRIES": unauthenticatedRegistries(ctx, asc),
 			"HW_VALIDATOR_REQUIREMENTS":   defaultControllerHardwareRequirements,
+			"ENABLE_DATA_COLLECTION":      "True",
```

Comment:
Does this need to be configurable? Or does the user do that in some other way?

Reply:
Discussed with Nick offline. There's another way we recommend users to opt-out and that's using the [telemetry opt-out](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/support/remote-health-monitoring-with-connected-clusters#opting-out-remote-health-reporting). Added a comment to clarify this.